src/lib.rs

```diff
@@ -127,8 +130,68 @@ pub fn preprocess(
     (res, cpp.warnings())
 }
 
+pub fn initialize_jit_module() -> Module<SimpleJITBackend> {
+    let mut flags_builder = cranelift::codegen::settings::builder();
+
+    // use debug assertions
+    flags_builder
+        .enable("enable_verifier")
+        .expect("enable_verifier should be a valid option");
+    // minimal optimizations
+    flags_builder
+        .set("opt_level", "speed")
+        .expect("opt_level: speed should be a valid option");
+    // don't emit call to __cranelift_probestack
+    flags_builder
+        .set("enable_probestack", "false")
+        .expect("enable_probestack should be a valid option");
+
+    let isa = cranelift::codegen::isa::lookup(arch::TARGET)
+        .unwrap_or_else(|_| utils::fatal(format!("platform not supported: {}", arch::TARGET), 5))
+        .finish(cranelift::codegen::settings::Flags::new(flags_builder));
+
+    let builder = SimpleJITBuilder::with_isa(isa, cranelift_module::default_libcall_names());
+    let module = Module::new(builder);
+    module
+}
+
+pub fn initialize_aot_module(name: String) -> Module<ObjectBackend> {
```

Comment:
These two functions are mostly the same, can you refactor them to have less duplicated code?

Reply:
Yes,I will do this.